odl/util/testutils.py

```diff
@@ -217,13 +217,13 @@ def is_subdict(subdict, dictionary):
     )
 
     skip_if_no_pywavelets = pytest.mark.skipif(
-        "not odl.trafos.wavelet.PYWAVELETS_AVAILABLE",
-        reason='Wavelet not available'
+        "not odl.trafos.PYWAVELETS_AVAILABLE",
```

Comment:
Writing this here but it is more general.

I feel that we mix `PYWAVELETS`, `PyWavelets` and `pywt` in a rather unorganized fashion. Is it obvious for users that `PYWAVELETS_AVAILABLE` correlates to `pywt` being installed? Perhaps we should rename this one to `PYWT_AVAILABLE`?